Ports/Another-World/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=Another-World
+useconfigure=true
+version=git
+depends="SDL2 zlib"
+workdir=Another-World-Bytecode-Interpreter-master
+configopts="-DCMAKE_TOOLCHAIN_FILE=${SERENITY_SOURCE_DIR}/Toolchain/CMake/CMakeToolchain.txt"
+files="https://github.com/fabiensanglard/Another-World-Bytecode-Interpreter/archive/refs/heads/master.zip master.zip 326de7622e5f83a83fce76e6032240157a9dde83c0d65319095c7e0b312af317"
+auth_type=sha256
+launcher_name="Another World"
+launcher_category=Games
+launcher_command="/opt/Another_World/raw --datapath=/opt/Another_World"
```

Comment:
Can we please use a dash for the folder name? Underscore is very uncommon across the system.

Reply:
Absolutely. I just pushed the change to the PR.

I took [Super-Mario](https://github.com/SerenityOS/serenity/blob/master/Ports/Super-Mario/package.sh) as an example, it is using underscore in the folder name.